### PR TITLE
Update ktor to 3.0.0

### PR DESCRIPTION
**Pull request by e5l**

### CODE REVIEW

**Comment by e5l** [comment]
Hey @JSMonk, I've bumped Ktor to the latest version. I'm going to setup ingress CORS proxy soon

File: build.gradle.kts [hunk 1]
```diff
@@ -1,4 +1,11 @@
 
+allprojects {
+    repositories {
+        maven("https://maven.pkg.jetbrains.space/public/p/ktor/eap")
+        google()
+        mavenCentral()
+    }
+}
```
Comment (terrakok) [inline build.gradle.kts:8]: I prefer to  configure it in the `settings.gradle.kts`:

```
dependencyResolutionManagement {
    repositories {
        maven("https://maven.pkg.jetbrains.space/public/p/ktor/eap")
        google()
        mavenCentral()
    }
}
```

File: README.md [hunk 2]
```diff
@@ -17,48 +17,31 @@ Once in a while, it connects to APIs to get the latest information about session
...
-This way iOS part itself is responsible only for specifying how the application looks like and how it represents changes requested by logic. Kotlin and Swift are highly interoperable, so from Swift, you can easily use all classes and tools defined in the common module. 
+Both Android and iOS applications are developed using Kotlin Multiplatform Mobile technology within a single codebase.
+The UI is implemented using Compose Multiplatform library, sharing the same code between platforms.
 
 ## How to build and run
 
-### Building the code
+### Pre-requisites
 
  * Make sure you have the Android SDK installed
```
Comment (terrakok) [inline README.md:29]: Do we need a Xcode pre-requisite add here?

**Review by terrakok — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (e5l) [reply README.md:29]: thanks, will add

**Pull request merged**